app/templates/main/index.html

```diff
@@ -4,21 +4,122 @@
+                        <label>Option ekkeke</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field one wide column">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+            </div>
+          </div>
+
+          <div id="advanced-search" class="option-group-advanced" style="display:none;">
+            <div class="descriptor-group">
+              <h5>DESCRIPTOR #2</h5>
+                <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+            </div>
+
+            <div class="descriptor-group">
+              <h5>DESCRIPTOR #3</h5>
+                <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+                    <div class="ui checkbox field">
+                      <input type="checkbox" name="option">
+                        <label>Option</label>
+                    </div>
+            </div>
+          </div>
+
+          <div id="advanced-filters-showhide">
+            <a href="javascript:showhide('advanced-search')" id="advanced-filters-showhide-link">
```

Comment:
instead of calling it like this, you should set a jQuery handler like
`$(#id).click(function to be called)`